### PR TITLE
[bug] 7375: Added Judge title in court-issued documents' title

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

File: web-api/run-local-migration.js [hunk 1]
```diff
@@ -4,16 +4,26 @@ const {
...
     region: 'us-east-1',
+    service: dynamo,
   });
 
   await documentClient
     .scan({
       ExclusiveStartKey: null,
       Segment: 0,
-      TableName: 'efcms-local',
+      TableName: process.env.SOURCE_TABLE,
```
Comment (rachaelparris) [inline web-api/run-local-migration.js:26]: I won't hold up the PR, just wondering why this should _not_ be hardcoded to `efcms-local` for running a migration locally?

File: shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeF.js [hunk 2]
```diff
@@ -20,11 +20,13 @@ CourtIssuedDocumentTypeF.prototype.init = function init(rawProps) {
   this.documentTitle = rawProps.documentTitle;
   this.documentType = rawProps.documentType;
   this.judge = rawProps.judge;
+  this.judgeWithTitle = rawProps.judgeWithTitle;
```
Comment (rachelschneiderman) [inline shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeF.js:23]: Should this be added to the schema?

File: web-api/run-local-migration.js (hunk 1 above)
Comment (sutt0n) [reply web-api/run-local-migration.js:26]: `npm run migrate:local` takes care of that.

File: shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeF.js (hunk 2 above)
Comment (sutt0n) [reply shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeF.js:23]: Done: https://github.com/flexion/ef-cms/pull/7402/commits/805dce89eb0542c39211190253b6d551c9e95fba

Comment (rachelschneiderman) [reply shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeF.js:23]: Thanks!

File: shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeB.js [hunk 3]
```diff
@@ -22,10 +22,12 @@ CourtIssuedDocumentTypeB.prototype.init = function init(rawProps) {
   this.documentType = rawProps.documentType;
   this.freeText = rawProps.freeText;
   this.judge = rawProps.judge;
+  this.judgeWithTitle = rawProps.judgeWithTitle;
```
Comment (rachelschneiderman) [inline shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeB.js:25]: Nit: Can this be added to the schema too at some point?

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**